configs/config.test.yml

```diff
@@ -18,12 +19,27 @@ experiment:
   mono-max-sentences-src: 100000
   mono-max-sentences-trg: 200000
 
-  bicleaner-threshold: 0.5
-
+  # split corpus to parallelize translation
   split-length: 100000
 
+  best-model: chrf
+
+  bicleaner:
+    default-threshold: 0.5
+    dataset-thresholds:
+      mtdata_neulab_tedtalksv1_train: 0.6
+
 training:
-  after-epochs: 1
+  s2s:
```

Comment:
Why are things stopping to train after just one epoch? This is inadequate for vast majority of language pairs. Is this just for testing purposes, as opposed to actually configuration used for training?

Reply:
This is a test config. Its purpose is to run it all end to end as quickly as possible to catch bugs. I'll add comments to make it clear.

Reply:
As mentioned in an earlier comment. I'd recommend using `after` instead of `after-epoch` for testing. Going through an entire epoch can take very long if you have lots of data to train on.

Reply:
updated configs to use `after`